dsp/window/window.go

```diff
@@ -30,14 +30,14 @@ func Rectangular(seq []float64) []float64 {
 // Sine window is a high-resolution window.
 //
 // The sequence weights are
-//  w[k] = sin(π*k/(N-1)),
+//  w[k] = sin(π*k/N),
```

Comment:
This description (and others) misses the `+ 1/2` offset.

Reply:
I had intended to show the general case and implement the specific case, but I'll fix this.

Reply:
If you did it on purpose, then it's fine as it is.

Reply:
Done.

Reply:
I think you are right though. Having it explicit helps explain the intervals comment and makes it easier for reviewers to see errors.